core/define/src/mill/define/ExternalModule.scala

```diff
@@ -22,3 +22,5 @@ abstract class ExternalModule(implicit
     Segments(millModuleEnclosing0.value.split('.').map(Segment.Label(_)).toIndexedSeq)
   }
 }
+
+class PackageExternalModule(val value: ExternalModule)
```

Comment:
Can we have some ScalaDoc here?

Reply:
Should we have the work "default" in the class name? It's IMHO a expected catch-word. We also have `TaskModule.defaultCommandName`. 

Maybe: `DefaultExternalModule` or `DefaultModule`. `DefaultPackageExternalModule` is too long.

Reply:
Renamed it `PackageDefaultExternalModule extends ExternalModule.Alias` and added some scaladoc, a bit long but not too bad. Still a one-liner

Reply:
I love the `Alias` idea! (Thinking if we somehow could allow it on all real modules, so we could give users the chance to lift their entry points (often burried deep in the hierarchy) into some other locations. This would reduce the need for local `justfile`s or `Makefile`s.)

The name is rather long honestly. We need a `Default` for a `Package` which is a  `Module`. Since it must be some instance, it doesn't matter whether it's a `ExternalModule` anymore, so the `External` is the superfluous part in that name.

Reply:
For now it's basically only useful for `ExternalModule`s, since the only time it comes into lay is when resolving external modules via `foo.bar.baz/`. It doesn't work for modules in `build.mill` for example, and we already have `def` for defining aliases inside your `build.mill` file

Reply:
the current documented workflow for aliasing external modules https://mill-build.org/mill/fundamentals/modules.html#_aliasing_external_modules